docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,164 @@
+* `snapshot_files_raw_hashes`, is a JSON record that contains the hashes of the files present in `_snapshot_additional_metadata.json` (the `Public state`, `Private data hashes`, `Transactions IDs`, and `Collection config history`).
+
+This another metadata file is a JSON record with the following two fields:
+
+* `snapshot_hash`, the hash of the file `_snapshot_signable_metadata.json` and can be treated as a hash of the snapshot.
+* `last_block_commit_hash`, which is included if the snapshot generating peer is equipped to compute the block commit hashes.
+
+Note that the file types explained here is a superset of all of the files that might be included in a snapshot. If admins find some of these file types missing in their snapshots (for example, the collection config history) this is not mean the snapshot is incomplete. The channel might not have any collections.
+
+## Joining a channel using a snapshot
+
+When joining a channel using the genesis block, a command similar to `peer channel join --blockpath mychannel.block` is issued. When joining the peer to the channel using a snapshot, issue a command similar to:
+
+```
+peer channel joinbysnapshot --snapshotPath <path to snapshot>
+```
+
+To verify that the peer has joined the channel successfully, issue a command similar to:
+
+```
+peer channel getinfo -c <name of channel joined by snapshot>
+```
+
+Additionally, if the peer has not already installed a chaincode being used on the channel, do so, and then issue a query. A successful return of data indicates that the peer has successfully joined using the snapshot. You can then install all of the chaincodes being used on the channel. If the snapshot is used by a new organization, and the channel contains the definitions for the chaincodes that are defined using the [new chaincode lifecycle](../chaincode_lifecycle.html), the new organization will need to approve the definition of these chaincodes can be invoked using the peer.
+
+## Try it out
+
+If you want to try out the ledger snapshotting process, you'll first need a network with a running channel. If you don't have a network, deploy the [test network](./test_network.html). This will create a network with two orgs, which both have a single peer, and an application channel.
+
+Select the peer you want to use to take the snapshot and follow the instructions above to take the snapshot. Then locate the snapshot on the your peer and copy it somewhere else on your filesystem.
+
+After you have taken the snapshot and copied it, follow the [Adding an Org to a Channel](./channel_update_tutorial.html) to add a new org to your network and application channel. When you reach the section where you are asked to [Join Org3 to the Channel](./channel_update_tutorial.html#join-org3-to-the-channel), instead of the `peer channel join -b mychannel.block` command, substitute `peer channel joinbysnapshot --snapshotPath <path to snapshot>` using the path to the snapshot on your filesystem.
```

Comment:
>After you have taken the snapshot and copied it, follow the [Adding an Org to a Channel]

The recommended way would be to take the snapshot after committing the new channel config that includes org3 (i.e., after completion of [Sign and Submit the Config Update] step) so, the snapshot includes the latest channel config. In other words, I would make the step for generating the snapshot as part of [Join Org3 to the Channel].